teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative('./person')
+
+class Teacher < Person
+  def initialize(*args, specialization, **kwargs)
+    super(*args, **kwargs)
+    @specialization = specialization
+  end
```

Comment:
Great job @sonyco-4u 🤝

- [ ] You may make sure your constructor looks something like the snippet below. It is also good to have descriptive variable names.
![constructor=specialization](https://user-images.githubusercontent.com/63917725/210644054-ab66779b-4375-487b-bd3f-199aae2ac209.PNG)